src/Sfa.Tl.ResultsAndCertification.Web/ViewModel/AdminDashboard/IndustryPlacement/AdminReviewChangesIndustryPlacementViewModel.cs

```diff
@@ -0,0 +1,159 @@
+﻿using Sfa.Tl.ResultsAndCertification.Common.Helpers;
+using Sfa.Tl.ResultsAndCertification.Web.Content.AdminDashboard;
+using Sfa.Tl.ResultsAndCertification.Web.Utilities.CustomValidations;
+using Sfa.Tl.ResultsAndCertification.Web.ViewComponents.BackLink;
+using Sfa.Tl.ResultsAndCertification.Web.ViewComponents.Summary.SummaryItem;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using IpStatus = Sfa.Tl.ResultsAndCertification.Common.Enum.IndustryPlacementStatus;
+
+namespace Sfa.Tl.ResultsAndCertification.Web.ViewModel.AdminDashboard.IndustryPlacement
+{
+    public class AdminReviewChangesIndustryPlacementViewModel
+    {
+        public int RegistrationPathwayId { get; set; }
```

Comment:
We could probably pack these properties into an independent class. Something like AdminLearnerDetailsViewModel. So we don't have to keep copying and pasting the same properties over and over again